tools/dynamic_framework_slicer/dynamic_framework_slicer.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# dynamic_framework_slicer copies the required dylibs for the given
+# binaries into the specified output path. This tool will also remove any
+# architecture slices which are not used by the given binaries, reducing binary
+# size.
+#
+# This script only runs on darwin and you must have Xcode installed.
+#
+# --out <file>      - the filename where the resultant binary should be placed.
+# --in <file>       - the filename of the input binary
+# <positional_args> - any positional argument is considered to be a
+#                              path to a binary to be processed 
```

Comment:
There is a trailing whitespace at the end of this line now.

Reply:
heh, doh.